### DISCUSSION TITLE
"Task list" case insensitive search?

### DISCUSSION THREAD

**Discussion by esbenboye** [GothenburgBitFactory/taskwarrior#3677] [Q&A] [answered]
I have a list of tasks and sometimes i want to filter it, to make it easier to find the id that i need to start. 
If i do a `task list <keyword>` then i get a list of tasks that contain that keyword.
My issue is that the keyword is case sensitive, which means there's a differnce between e.g. "git" and "Git". 

Is there a setting somewhere that i can set/change to make the `task list` filtering case insensitive?

**Comment by smemsh** [comment]
check out "man taskrc" and search for "sensitive"

**Reply by felixschurk** [reply, marked as answer]
The setting is documented, as mentioned in the manual. 

You want to add `search.case.sensitive=no` to your `.taskrc`.

**Comment by esbenboye** [comment]
@smemsh @felixschurk 
Thank you for replies - It works as intended!
